MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -127,23 +130,30 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
             return
         }
         if !requestParamValue.content.isEmpty {
-            hostAppMessageDelegate?.shareContent(info: MiniAppShareContent(messageContent: requestParamValue.content)) { (result) in
-                switch result {
-                case .success:
-                    self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callbackId, response: "SUCCESS")
-                case .failure(let error):
-                    if !error.localizedDescription.isEmpty {
-                        self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: error.localizedDescription)
-                        return
-                    }
-                    self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: getMiniAppErrorMessage(MiniAppErrorType.unknownError))
-                }
+            let info = MiniAppShareContent(messageContent: requestParamValue.content)
+            let hamDelegate = self.hostAppMessageDelegate ?? self
```

Comment:
I see the `hostAppMessageDelegate` value is not optional, why do we still trying to `self` ?

Reply:
see my previous comment